guides/index_lifecycle.md

```diff
@@ -3,23 +3,13 @@ This guide covers OpenSearch JavaScript Client API actions for Index Lifecycle.
 
 ## Setup
 
-In this guide, we will need an OpenSearch cluster with more than one node. Let's use the sample [docker-compose.yml](https://opensearch.org/samples/docker-compose.yml) to start a cluster with two nodes. The cluster's API will be available at `localhost:9200` with basic authentication enabled with default username and password of `admin:admin`.
-
-To start the cluster, run the following command:
-
-```bash
-cd /path/to/docker-compose.yml
-docker-compose up -d
-```
-
-Let's create a client instance to access this cluster:
+Let's create a client instance to access an OpenSearch cluster:
```

Comment:
Should we skip mentioning how to start a cluster here?

Reply:
I think we should. These guides should focus on OpenSearch features, not how to setup a cluster. We can have another guide to cover how to set up a cluster, but that's already covered in the Readme and especially the OpenSearch website itself.